### ISSUE TITLE
Continuous deployment for xdai.io

### ISSUE THREAD

**Issue by TimDaub** [austintgriffith/burner-wallet#203]
## Scope
- Currently the burner wallet relies on @austintgriffith to be deployed to e.g. xdai.io.
- As we'd like to ship to production as often as possible, having continuous integration would be nice.
- The LeapDAO team has already done this with their burner, so we could just copy what they're doing: https://github.com/leapdao/burner-wallet/pull/144
- Note that we might not want to deploy everything in master directly into production. As the code base is handling actual people's money, I'd propose we do as following:
  - Have a branch that deploys a burner-wallet to test.xdai.io with e.g. Rinkeby support (maybe that's master). Pro-tip for making Exchange.js work on Rinkeby: [Uniswap has contracts deployed on Rinkeby](https://docs.uniswap.io/frontend-integration/connect-to-uniswap#factory-address)
  - Have a branch that deploys to xdai.io (e.g. "production")

## Deliverables
- Implement continuous deployment for the burner-wallet, a staging domain and a production domain
- Remove the now outdated scripts used for deployment

## Notes

- This issue can only be resolved by someone having access to the hosting provider @austintgriffith uses. Make sure you talk to him first before starting this issue.
- Consider that #208 and this issue might be related or dependent on each other.

## Roles
bounty gardener: @TimDaub / 10%
bounty worker: name / 75%
bounty reviewer: name / 15%

**Comment by dmihal** [comment]
This is related to #208, regarding which branch is used for what.

My suggestion would be master is for production, and we have staging and develop as well with their own deployments as well. 

I'd also suggest each PR gets an automatic "review app" deployed, assuming there's funding for that many instances.

**Comment by rovidev95** [comment]
Interested in this bounty but **blocked without hosting/deploy credentials** for xdai.io (per issue notes). @TimDaub — if you can grant deploy access or confirm Firebase/hosting workflow, I'll implement CD following leapdao/burner-wallet#144 pattern. Branching proposal in docs/BRANCHING.md (PR #311).

**Comment by rovidev95** [comment]
Partial delivery: **#319** — deploy workflow + docs. Build job works now; deploy needs your AWS secrets (see docs/DEPLOY.md). /claim rovidev95@gmail.com @TimDaub
